Resume_Project.html

```diff
@@ -6,20 +6,30 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="Resume_Project.css">
+    <script src="resume.js"></script>
+ 
```

Comment:
just fyi, you'll see the script tag in the head tag or you'll see it as the last tag before the end of the body. both work.